app/src/components/existing-request/nr-approved-gray-box.vue

```diff
@@ -242,6 +242,14 @@ export default class NrApprovedGrayBox extends Mixins(CommonMixin) {
   @Prop({ default: false }) readonly disabled!: boolean
 
   @Getter getNr!: Partial<NameRequestI>
+  @Getter getIsLearBusiness!: boolean
+
+  isBusinesCheckDone = false
+
+  /** Called when component is mounted. */
+  mounted () {
+    this.checkBusiness()
```

Comment:
Technically this won't work because it's async and you're not awaiting it. In other words, mounted() will complete before checkBusiness() is done.

Just FYI since my other comment proposed a cleanup that will obsolete this comment.